### ISSUE TITLE
(phi0448.phi001.perseus-lat2:2) typos

### ISSUE THREAD

**Issue by johnsonb2** [PerseusDL/canonical-latinLit#350]
2.22.1: "delectusque" should be "deiectusque"?
2.24.4: "tunditores" should be "funditores"?

**Comment by lcerrato** [comment]
delectusque is in the print (although marked as spurious)
https://archive.org/details/ciulicaesarisco00caesgoog/page/n166/mode/2up

funditores is the correct reading

**Comment by lcerrato** [comment]
25.1: "prolectus" should be "profectus"
30.4: "mallibus" should be "manibus"
33.2: "accensus" should be "ascensus"

**Comment by lcerrato** [comment]
https://archive.org/details/ciulicaesarisco00caesgoog/page/n168/mode/2up
https://archive.org/details/ciulicaesarisco00caesgoog/page/n174/mode/2up
https://archive.org/details/ciulicaesarisco00caesgoog/page/n176/mode/2up

**Comment by lcerrato** [comment]
leave open as standing issue for future typos

**Comment by johnsonb2** [comment]
4.2.3: "adsue fecerunt" should be "adsuefecerunt"
4.4.5: "regiones" should be "regionesque"
4.6.2: "quas" should be "quae"
4.14.5: "eum" should be "cum"
4.15.1: "eum" should be "cum"
4.15.2: "eum" should be "cum"
4.17.3: remove period after "sesquipedalia"
4.23.5: remove "D" from "ad nutum et ad tempus D omnes..."
English:
4.17: "dike a stake" should be "like a stake"
4.21: "Caius Volusenus" should be "Gaius Volusenus"? (if this is not intentional)

**Comment by lcerrato** [comment]
NB Two of the above are in phi0448.phi001.perseus-eng2.xml

**Comment by lcerrato** [comment]
Caius is in print as a variant. (stet)

**Comment by lcerrato** [comment]
fixed most recent report

**Comment by johnsonb2** [comment]
5.5 English: "There he discovers that **forty** ships..." translating LX as "forty" and not "sixty" as on page 239 here: [https://archive.org/details/gallicwar00caes/page/238/mode/2up](url)

**Comment by lcerrato** [comment]
@johnsonb2 
Both LX and XL are attested. It is not uncommon for the Perseus English to not match the original language since we do not always use translations from the same volume as the primary editions.

See https://www.google.com/books/edition/C_Julii_Cæsaris_commentarii_de_bello_Ga/WL1XAAAAcAAJ?hl=en&gbpv=1&dq=%22Ibi+cognoscit+XL+naves%22&pg=PA103&printsec=frontcover

A note can be added to the English to avoid the confusion, but this is a variant.

**Comment by johnsonb2** [comment]
OK, good to know. Is that the source of the English translation?

**Comment by lcerrato** [comment]
No, the translation is linked via the Perseus Catalog (the left hand sidebar)
https://catalog.perseus.org/catalog/urn:cts:latinLit:phi0448.phi001.perseus-eng1

It says in the front matter it is Oudendorf. Here is one printing:
https://archive.org/details/workslatin1819cj00caes/page/106/mode/2up

**Comment by johnsonb2** [comment]
5.6.1: "magri" should be "magni"
5.11.9: "praefeceraut" should be "praefecerant"
5.13.1: "mila" should be "milia"(?)
5.24.7: "im" should be "in"
5.25.2: "onera" should be "opera"
5.25 (English) "...many even of his own state being openly promoters [of that 
act]" missing period
5.26.4: "Tum suo more conclamaverunt, uti aliqui ex nostris ad colloquium 
prodiret: ...": The verb "prodiret" is in the singular, but the English 
translation is in the plural ("prodirent variant" from Oudendorf?).
5.37 (English): missing period at end
5.48 (English): "Is by chance stuck in a tower..."

**Comment by lcerrato** [comment]
note phi0448.phi001.perseus-eng2 also referenced here

**Comment by johnsonb2** [comment]
From now on should I open a separate issue for phi0448.phi001.perseus-eng2?

**Comment by lcerrato** [comment]
If there are going to be a substantial number in the other text, then it should have its own issue or I'll change the name of this issue.

**Comment by johnsonb2** [comment]
7.14.4: "deligi" should be "deleri"(?)
7.16.3: "Omnis" should be "Omnes"
7.19: "Gaesar" should be "Caesar"
7.43: "fatrumque" should be "fratrumque"(?)
